esmvaltool/recipes/recipe_ecs_scatter_cmip6.yml

```diff
@@ -0,0 +1,303 @@
+# recipe_ecs_scatter.yml
+---
+documentation:
+  description: |
+    Calculates equilibrium climate sensitivity (ECS) versus
+    1) southern ITCZ index, similar to fig. 2 from Tian (2015)
+    2) lower tropospheric mixing index (LTMI), similar to fig. 5 from
+       Sherwood et al. (2014)
+    3) tropical mid-tropospheric humidity asymmetry index, similar to fig. 4
+       from Tian (2015)
+    4) covariance of shortwave cloud reflection (Brient and Schneider, 2016)
+    5) climatological Hadley cell extent (Lipat et al., 2017)
+
+    ***************************************************************************
+    Note: this recipe requires pre-calulation of the equilirium climate
+          sensitivites (ECS) for all models. The ECS values are calculated
+          with recipe_ecs.yml. The netcdf file containing the ECS values
+          (path and filename) is specified by diag_script_info@ecs_file.
+          Alternatively, the netcdf file containing the ECS values can be
+          generated with the cdl-script
+          $diag_scripts/emergent_constraints/ecs_cmip.cdl (recommended method):
+          1) save script given at the end of this namelist as ecs_cmip.cdl
+          2) run command: ncgen -o ecs_cmip.nc ecs_cmip.cdl
+          3) copy ecs_cmip.nc to directory given by diag_script_info@ecs_file
+             (e.g. $diag_scripts/emergent_constraints/ecs_cmip.nc)
+    ***************************************************************************
+
+  authors:
+    - laue_ax
+
+  maintainer:
+    - laue_ax
+
+  references:
+    - brient16jclim
+    - lipat17grl
+    - sherwood14nat
+    - tian15grl
+
+  projects:
+    - crescendo
+
+
+preprocessors:
+  pp500:
+    extract_levels:
+      levels: 50000
+      scheme: linear
+
+  pp_ltmi_0:
+    extract_levels:
+      levels: [85000, 70000, 60000, 50000, 40000]
+      scheme: linear
+
+  pp_ltmi_1:
+    extract_levels:
+      levels: [85000, 70000]
+      scheme: linear
+
+
+diagnostics:
+
+  itczidx:
+    description: southern ITCZ index
+    themes:
+      - EC
+    realms:
+      - atmos
+    variables:
+      pr:
+        reference_dataset: TRMM-L3
+        start_year: 1998
+        end_year: 2013
+        exp: historical
+        mip: Amon
+        project: CMIP6
+        additional_datasets:
+          - {dataset: TRMM-L3, project: obs4mips, level: v7, version: 7A, tier: 1}
+          - {dataset: IPSL-CM6A-LR, grid: gr, ensemble: r19i1p1f1}
+          - {dataset: MIROC6, grid: gn, ensemble: r1i1p1f1}
+          - {dataset: GISS-E2-1-G, grid: gn, ensemble: r1i1p1f1}
+#          - {dataset: CNRM-ESM2-1, grid: gr, ensemble: r1i1p1f2}
```

Comment:
Why are there so many commented out datasets in this recipe? Commented out stuff should either be uncommented or removed before merging.

Reply:
Incomplete datasets have been removed from the recipe.